spec/requests/api/v1/inventories_controller_spec.rb

```diff
@@ -92,4 +92,97 @@
       expect(hash_body).to match({})
     end
   end
+
+  describe 'POST inventory, #create' do
+    before :all do
+      @valid_headers['Content-Type'] = 'application/json'
+      @request_body = { data:
+                         { type: 'inventories',
+                           attributes: {
+                             name: Faker::Name.unique.name
+                           } } }
+    end
+
+    it 'Response with correct inventory' do
+      hash_body = nil
+      post api_v1_team_inventories_path(
+        team_id: @teams.first.id
+      ), params: @request_body.to_json, headers: @valid_headers
+      expect(response).to have_http_status 201
+      expect { hash_body = json }.not_to raise_exception
+      expect(hash_body[:data]).to match(
+        ActiveModelSerializers::SerializableResource
```

Comment:
Can you change this check to:
```ruby
expect(hash_body[:data]).to match(
  ActiveModelSerializers::SerializableResource
    .new(Repository.last,
         serializer: Api::V1::InventorySerializer)
    .as_json[:data]
)
```

Reply:
https://github.com/biosistemika/scinote-web/pull/1295/commits/088e06cae732b18b8e9eefb5c94790c3167ed644